docs/administrator/migration/migrate-in-batch.md

```diff
@@ -86,12 +88,84 @@ spec:
   schedulerName: default-scheduler
 ```
 
+#### Eg3. migrate all resources related to a specific application.
+
+Assuming a specific application consists of `deployment/nginx` and `service/nginx-svc`, 
+and you want to migrate the resources related to this application from the `member1` cluster to Karmada, 
+you need to apply the following configuration:
+
+```yaml
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: nginx-pp
+spec:
+  conflictResolution: Overwrite
+  placement:
+    clusterAffinity:
+      clusterNames:
+      - member1
+  priority: 0
+  resourceSelectors:
+  - apiVersion: apps/v1
+    kind: Deployment
+    name: nginx
+  - apiVersion: v1
+    kind: Service
+    name: nginx-svc
+  schedulerName: default-scheduler
```

Comment:
```suggestion
apiVersion: policy.karmada.io/v1alpha1
kind: PropagationPolicy
metadata:
  name: nginx-pp
spec:
  conflictResolution: Overwrite
  placement:
    clusterAffinity:
      clusterNames:
      - member1
  resourceSelectors:
  - apiVersion: apps/v1
    kind: Deployment
    name: nginx
  - apiVersion: v1
    kind: Service
    name: nginx-svc
```
The `priority` and `schedulerName` are optional during the migration. We'd better remove them to avoid misleading.